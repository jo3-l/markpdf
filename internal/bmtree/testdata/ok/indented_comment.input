p1. foo
    # bar
    p3. baz