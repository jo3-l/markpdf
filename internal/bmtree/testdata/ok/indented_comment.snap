- "foo" @ p1
	- "baz" @ p3